src/vm/dwbucketmanager.hpp

```diff
@@ -943,30 +943,28 @@ DWORD BaseBucketParamsManager::GetILOffset()
     DWORD nativeOffset = 0;
     DWORD ilOffset = MAXDWORD;
 
-    EECodeInfo codeInfo(m_faultingPc);
-    if (codeInfo.IsValid())
-    {
-        nativeOffset = codeInfo.GetRelOffset();
-        _ASSERTE(m_pFaultingMD == codeInfo.GetMethodDesc());
-    }
-
-    if (m_pFaultingMD)
+    if (g_pDebugInterface != NULL)
```

Comment:
Maybe add a comment that you aren't aware of any current scenario where g_pDebugInterface is NULL, but that we might do so in the future?

Reply:
Actually I think there should be an assert that g_pDebugInterface != NULL, or simply revert the code. As-is this feels like a partial implementation. Anyone who enables g_pDebugInterface = NULL in the future needs to explicitly reconsider this code. Without more changes all the IL mappings will have been needlessly lost and that probably isn't what the future implementer will want to occur.